programs/bpf_loader/src/syscalls.rs

```diff
@@ -53,6 +53,9 @@ use thiserror::Error as ThisError;
 /// Maximum signers
 pub const MAX_SIGNERS: usize = 16;
 
+/// Maximum size return data
+pub const MAX_RETURN_DATA: u64 = 1024;
```

Comment:
Can this be pulled from the SDK so it is defined in one place?

Reply:
Yes, I missed that. Thanks for catching this.

Reply:
Fixed